src/sentry/static/sentry/app/views/organizationIncidents/details/body.jsx

```diff
@@ -18,7 +18,7 @@ const TABS = {
   related: {name: t('Related incidents'), component: RelatedIncidents},
 };
 
-export default class Incidents extends React.Component {
+export default class IncidentBody extends React.Component {
```

Comment:
Should this be IncidentDetailsBody to more closely match the file name and default export?

Reply:
What about `DetailsBody` (or maybe just `Body`) -- the main view is `organizationIncidents` pluralized

Reply:
@lynnagara do you have any thoughts here since we went through the same thing

Reply:
That seems fine, there's no reason to repeat Incidents everywhere 😀